build.gradle

```diff
@@ -16,7 +16,7 @@ ext {
 }
 
 group "com.github.ironman19933"
-version "3.1.2"
+version "3.1.1"
```

Comment:
3.1.3, it version should be only incremental

Reply:
Done